docs/howto_firewall.md

```diff
@@ -1,33 +1,40 @@
 # Firewall Configuration
-MicroShift does not require a firewall to run. The following ports should be considered for MicroShift if a firewall is enabled.
+MicroShift does not require a firewall for normal operation. If a firewall is enabled, it should be configured according to the requirements in this document.
 
-|Port(s)    |Protocol(s)|Description|
-|:----------|:----------|:----------|
-|80         |TCP        | HTTP port used to serve applications through the OpenShift router |
-|443        |TCP        | HTTPS port used to serve applications through the OpenShift router |
-|6443       |TCP        | HTTPS API port for the MicroShift API |
-|5353       |UDP        | mDNS service to respond for OpenShift route mDNS hosts |
-|30000-32767|TCP/UDP    | Port range reserved for NodePort type of services, can be used to expose applications on the LAN |
+It is mandatory to allow MicroShift pods the access to the internal CoreDNS and API servers.
+> Users may choose to change the pod IP range to be different from the default `10.42.0.0/16` setting. This must be reflected in the firewall configuration.
```

Comment:
:+1

Reply:
Sorry, @mangelajo - anything I should change here?

Reply:
nothing just that i liked the addition, but the emoji didn't work :)